packages/app/src/components/HyperJson.tsx

```diff
@@ -142,6 +145,17 @@ const Line = React.memo(
     // mounting it for potentially hundreds of lines
     const { ref, hovered } = useHover<HTMLDivElement>();
 
+    const isCollapsibleString =
+      isString(value) &&
+      (value.split('\n').length > 1 ||
+        value.length > STRING_COLLAPSE_THRESHOLD);
+    const collapseString = isCollapsibleString
+      ? (value.split('\n').length > 1 ? value.split('\n')[0] : value).substring(
+          0,
+          COLLAPSE_STRING_SIZE,
+        ) + '...'
+      : '';
+
```

Comment:
It would be neat if we could use mantine's Spoiler component for this: https://mantine.dev/core/spoiler/

Reply:
Probably not necessary for now, but good to keep in mind.